packages/mdc-tab/index.js

```diff
@@ -63,6 +66,7 @@ class MDCTab extends MDCComponent {
   initialize(
     rippleFactory = (el, foundation) => new MDCRipple(el, foundation),
     tabIndicatorFactory = (el) => new MDCTabIndicator(el)) {
+    this.id = this.root_.id;
```

Comment:
Is the element's id guaranteed to be set by the time this runs?

Reply:
It is guaranteed to be set. I based this off chips which is working appropriately.

https://github.com/material-components/material-components-web/blob/6739ea0e4e7aa2cd0c93b829bade501a05eb3224/packages/mdc-chips/chip/index.js#L71